packages/access-api/src/service/index.js

```diff
@@ -46,21 +87,29 @@ export function service(ctx) {
               audience: DID.parse(capability.with),
               with: ctx.signer.did(),
               lifetimeInSeconds: 60 * 10,
+              nb: {
+                identity: email,
+              },
               proofs: [
                 await Account.recover.delegate({
                   audience: ctx.signer,
                   issuer: ctx.signer,
-                  lifetimeInSeconds: 60 * 1000,
+                  expiration: Infinity,
                   with: ctx.signer.did(),
+                  nb: {
+                    identity: 'mailto:*',
+                  },
                 }),
               ],
             })
             .delegate()
 
           const encoded = await delegationToString(inv)
+          const url = `${ctx.url.protocol}//${ctx.url.host}/validate-email?ucan=${encoded}&mode=recover`
+
           // For testing
           if (ctx.config.ENV === 'test') {
-            return encoded
+            return url
```

Comment:
Only return in test?

Reply:
yes just for tests so we can process the delegation without dealing with emails.